TypeScript.py

```diff
@@ -1,19 +1,13 @@
 import sys
+import os
 
-# Sublime/Python 2 & 3 differ in the name of this module, thus package import
-# needs to be handled slightly differently
-if sys.version_info < (3, 0):
-    from libs import *
-    from libs.reference import *
-    from libs.view_helpers import *
-    from listeners import *
-    from commands import *
-else:
-    from .libs import *
-    from .libs.reference import *
-    from .libs.view_helpers import *
-    from .listeners import *
-    from .commands import *
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Not familiar with the above discussion, but if you do decide to continue using this approach, use [`os.path.join`](https://docs.python.org/2/library/os.path.html#os.path.join) instead of `append`.